jvcproj/README.md

```diff
@@ -0,0 +1,251 @@
+# JVC D-ILA Control
+
+With this plugin you can control JVC D-ILA projectors over TCP by using the "JVC External Control Command Communication Specification" and transfer gammatables generated with jvcprojectortools.
+
+## Notes
+I use the plugin to control my JVC DLA- X5000 and transfer raw gammatables generated with jvcprojectortools by arvehj. Many thanks to arvehjs groundwork so i could have a look in the communication procedure!
+You can use all operation commands given in the "JVC External Control Command Communication Specification". Wrong or unknown commands will be ignored by the plugin and the projector. Most of the commands won`t be acknowledged when the projector has no input signal. Request commands are not yet supported.
+The gamma table configuration files can be raw gamma table files (loaded from the projector with jvcprojectortools) or the configuration files that are generated with jvcprojectortools when you save your parameters. When using these files only the deposited raw gamma data ("table"- data) is needed and will be transfered.
+
+The plugin should work from the projector generation X3/X7/X9 up to X5900/X7900/X9900 and surely for upcoming generations, too.
+
+Please use [this thread for support, questions, feedback etc.](https://knx-user-forum.de/forum/supportforen/smarthome-py/1188479-plugin-steuerung-von-jvc-d-ila-projektoren)
+
+## Configuration
+
+### plugin.yaml
+
+```yaml
+jvcproj:
+    class_name: JVC_DILA_Control
+    class_path: plugins.jvcproj
+    host: 1.1.1.1 # host adress of the projector
+    port: ... # optional, default port is 20554
+    gammaconf_dir: ... # optional, location gamma table configuration files
+```
+
+#### Attributes
+  * `host`: the host adress of the projector
+  * `port`: the port the projector listens to. Default is 20554
+  * `gammaconf_dir`: location where the gammatable configuration files are saved. Default is '/usr/local/smarthome/etc/jvcproj/'
+
+### items.yaml
+
+#### operation commands
+The plugin uses the raw hex operation commands. A list of all commands (depending on the projectors series) can be found by looking at the JVC support homepage or by searching for JVC projector RS-232 command lists.
+Here is a small example for an operation command "Power On" (21 89 01 50 57 31 0A):
+
+  * `21`: Header - an operation command always start with 21 (ASCII: "!")
+  * `89 01`: Unit ID - is fixed at 89 01 for all models
+  * `50 57`: Command - varies depending on the command. In this example 50 57 (ASCII: "PW")
+  * `30`: Data - This is the value to apply to the command. Using the Power example above, the data value for On is 31 (ASCII: "1"). Length varies depending on command (not always 1 byte)!
+  * `0A`: End - This signifies the end of the command and is fixed at 0A for all models.
+
+### Example:
+
+```yaml
+jvcproj:
+
+    testcmd:
+    # command NULL
+        type: bool
+        visu_acl: rw
+        jvcproj_cmd: 21 89 01 00 00 0A
+        enforce_updates: 'True'
```

Comment:
Do you really need these single quotes around "True"?

Reply:
No, i do not think so! I think these quotes exist in my items.conf since the beginning of smarthome.py! I will change it!